SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -595,6 +596,13 @@ Buffer {
 		^[\b_query, bufnum]
 	}
 
+	sampleRate_ { arg newValue;
```

Comment:
a good method name would be `setSampleRate`. This would make clear it is a bit more than just setting the instance variable. In general, this kind of thing somtimes done one way, sometimes another, but here we need to make sure it doesn't create unwanted network messages.

Reply:
I just suggested the same thing :) https://github.com/supercollider/supercollider/pull/6710#issuecomment-2790760340